joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -1008,6 +1008,15 @@ controller_interface::CallbackReturn JointTrajectoryController::on_activate(
     read_state_from_state_interfaces(state_current_);
     read_state_from_state_interfaces(last_commanded_state_);
   }
+<<<<<<< HEAD
+=======
+  // reset/zero out all of the PID's (The integral term is not retained and reset to zero)
+  for (auto & pid : pids_)
+  {
+    pid->reset();
+  }
+  last_commanded_time_ = rclcpp::Time();
```

Comment:
Sorry I don't have permission to push a fix here. I don't think this term is needed here and then this is good to go.